Detections/Corelight/CorelightPossibleWebshell.yaml

```diff
@@ -0,0 +1,32 @@
+id: f3245aa1-1ca1-471c-a0b7-97ea6b791d5d
+name: Corelight - Possible Webshell
+description: |
+  'Detects post requests to unusual extensions.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: Corelight
+    dataTypes:
+      - Corelight_CL

```

Comment:
If we are using parser in query datatype should be parser name. So could you please update the datatype name to Corelight.

Reply:
updated